.travis.yml

```diff
@@ -5,3 +5,6 @@ go:
 
 script:
   - env GO111MODULE=on go build
+  - echo "$DOCKER_PASSWORD" | docker login docker.pkg.github.com --username "$DOCKER_USERNAME" --password-stdin
```

Comment:
This probably is just feeding the password value into the second command, but can we double check the echo does not log the password to the console?

Reply:
Yeah, when I looked at Travis logs, the password isn't logged. However, there are still improvements we can make by figuring out how to use a credential store.